internal/conversation/conversation.go

```
@@ -848,37 +847,32 @@ func (c *Manager) UpdateConversationTeamAssignee(uuid string, teamID int, actor
 
-	// Assignment successful, any errors now are non-critical and can be ignored by returning nil.
+	// Assignment succeeded, errors after this point are non-critical and must not skip the remaining side effects.
 	if err := c.RecordAssigneeTeamChange(uuid, teamID, actor); err != nil {
-		return nil
+		c.lo.Error("error recording team assignee change", "uuid", uuid, "error", err)
 	}
 
 	// Team changed?
 	if previousAssignedTeamID != teamID {
 		// Remove assigned user if team has changed.
 		c.RemoveConversationAssignee(uuid, models.AssigneeTypeUser, actor)
```

golangci-lint (2.12.2)

[error] Error return value of `c.RemoveConversationAssignee` is not checked

(errcheck)